### PR TITLE
Removes most trailing zeroes after decimal point

### PR DESCRIPTION
**Pull request by ZWRh3**
## About the PR 
removes (most) trailing zeroes, tries to make more stuff use defines

NOTE: its most not all as trying to basically manually replace them all is gonna make me go insane

## Why's this needed? 
cleanliness

### CODE REVIEW

**Comment by ZWRh3** [comment]
any tips on finding the rest?

File: code/procs/elecflash.dm [hunk 1]
```diff
@@ -117,4 +117,4 @@ var/global/mutable_appearance/elecflash_ma = null
 			src.do_disorient(stamina_damage = 15 + power * 8, weakened = 0, stunned = 0, paralysis = 0, disorient = (power * (0.5 SECONDS)), remove_stamina_below_zero = 0, target_type = DISORIENT_BODY)
 		else
 			src.do_disorient(stamina_damage = 15 + power * 8, weakened = 1 SECONDS + (power * (0.1 SECONDS)), stunned = 0, paralysis = 0, disorient = (power * (0.5 SECONDS)), remove_stamina_below_zero = 0, target_type = DISORIENT_BODY)
-		src.TakeDamage("chest", 0, rand(0,1.00) * power * 0.2, damage_type=DAMAGE_BURN)
+		src.TakeDamage("chest", 0, rand(0,1.0) * power * 0.2, damage_type=DAMAGE_BURN)
```
Comment (Luxizzle) [inline code/procs/elecflash.dm:120]:
```suggestion
		src.TakeDamage("chest", 0, rand(0,1) * power * 0.2, damage_type=DAMAGE_BURN)
```

**Review by Luxizzle — CHANGES_REQUESTED** [review]
Why would you do this to yourself?

Quick scan, but looks good! But damn, this is used way more than I expected.

**Comment by Josephin23** [comment]
> Why would you do this to yourself?
> 
> Quick scan, but looks good! But damn, this is used way more than I expected.

most of this was done with regex, idk how you'd do all of this manually

**Comment by ZWRh3** [comment]
> > Why would you do this to yourself?
> > Quick scan, but looks good! But damn, this is used way more than I expected.
> 
> most of this was done with regex, idk how you'd do all of this manually

while i did search for several strings using regex to semi automate it i still manually double checked stuff, this took hours to do

**Comment by ZeWaka** [comment]
ur fault if something breaks

**Comment by ZWRh3** [comment]
> ur fault if something breaks

this may either stop your concerns or make them worse but: i manually double checked it all

**Pull request merged**